app/main.py

```diff
@@ -1,6 +1,40 @@
+import requests
+
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    params = {
+        "key": os.getenv("WEATHER_API_KEY"),
+    }
+
+    get_weather_in_paris_url = (
+        "http://api.weatherapi.com/v1//"
+        "current.json?q=Paris"
```

Comment:
The URL string concatenation seems to have an extra slash in `"http://api.weatherapi.com/v1//"`. It should be `"http://api.weatherapi.com/v1/"` to form a valid URL.